docs/pywbemcligeneraloptions.rst

```diff
@@ -61,166 +70,470 @@ command, general the options redefinition's are not retained between command
+     used if no namespace is included with each command,
+   * several connection security general options:
+
+     * The :ref:`--user general option`,  defines the WBEM server user name
+     * The :ref:`--password general option`, defining the password for the WBEM
+       server user
+     * The :ref:`--no-verify general option` that defines whether the client verifies
+       certificates received from the WBEM server,
+     * The :ref:`--certfile general option`, client certificate file,
+     * The :ref:`--keyfile general option`,  client key file,
+     * The :ref:`--ca-certs general option`), a collection of certificates against
+       which certificates received from the WBEM server are verified.
+
+   * The :ref:`--timeout general option` that defines the client side connection
+     timeout
+
+2. Define a mock-server. A mock server see :ref:`Mock WBEM server support`
+   substitutes a local mock WBEMConnection for a WBEM server and allows
+   testing or demonstrating pywbemcli without having access to a real WBEM
+   server.
+
+3. Define the characteristics for either a WBEM server or mock WBEM server with
+   pywbemcli and add the data to a pywbemcli :term:`connections file`.
+   pywbemcli can access the server data by simply defining the name on with the
+   pywbem call.
+
+
+.. _`Controlling operation behavior and monitoring operations`:
+
+Controlling operation behavior and monitoring operations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Several options control the behavior and monitoring of the operations executed
+against the WBEM server.
+
+Since the DMTF specifications allow alternative forms of some operations, pywbemcli
+implements this flexibility and controls the choice of either the pull operations
+or the traditional operations through the :ref:`--use-pull general option`. With
+this option the user can chose to use either type of operation if that
+operation is available on the WBEM server. See :ref:`Pywbemcli and
+the DMTF pull operations` for more information on pull operations.
+
+Since the pull operations include the ability to select the maximum size of
+returned chunks of data, the :ref:`--pull-max-cnt general option` can be used
+to control response chunk sizes.
+
+In many cases it is important to the user to be able to monitor details of the
+operations executed against the WBEM server, either the APIs executed in pywbem,
+or the HTTP/HTML requests and responses and the time statistics for these
+operations.
+
+The :ref:`--log general option` provides the capability to log information about
+this flow including:
+
+* The API calls and responses/exceptions executed by pywbem.
+* the HTTP requests and responses that pass between pywbemcli and the WBEM Server
+
+The :ref:`--log general option` configures the logging including what is logged
+and the destination for the log output.
+
+Thus, TODO eample
+
+.. _`Controlling result output formats`:
```

Comment:
We also have 3.5. "Output formats". Do we really need to repeat most of that here?

Reply:
TODO

Reply:
Removed almost all of the Controlling result output formats section and put into section 3.5.

DONE

Reply:
Reduced to just pointing at the "output formats' section